kong/runloop/handler.lua

```diff
@@ -1288,6 +1293,13 @@ return {
       var.upstream_x_forwarded_path   = forwarded_path
       var.upstream_x_forwarded_prefix = forwarded_prefix
 
+      do
+        local req_via = get_header(constants.HEADERS.VIA, ctx)
+        local kong_inbound_via = var.server_protocol .. " " .. SERVER_HEADER
+        var.upstream_via = req_via and req_via .. "," .. kong_inbound_via  or
+                          kong_inbound_via
```

Comment:
It seems that there is no space ` ` inserted  between request via value and our kong_inbound_via. From RFC, we could find an example:

```
Via: 1.0 fred, 1.1 p.example.net
```

```suggestion
        var.upstream_via = req_via and req_via .. ", " .. kong_inbound_via  or
                          kong_inbound_via
```

Reply:
The space after the `,` is **optional** according to RFC. But adding a space make it more readable. Will accept this suggestion.

Reply:
Done.